nf-example-topology/packagevariantset-smf-regional.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: PackageVariantSet
+metadata:
+  name: nf-example-free5g-smf-regional
+spec:
+  upstream:
+    package:
+      repo: https://github.com/nephio-project/free5gc-packages
+      name: pkg-example-smf-bp
+    revision: v1
+  targets:
+  - repositories:
+      nephio.org/site-type: regional
```

Comment:
"regional" and "edge" is something I made up. If we think there are better, more telco-legit concepts, we should do that. Same goes for `site-type`, etc. We should think about what a reasonable "label schema" might look like that many users would be able to leverage. Also, what we need to *exercise*. For example, if we add a bunch of other NFs, how do we contruct our label keys and values such that we can land NFs where we want, and then scale those (by adding more clusters) in the right directions?

Reply:
I think "regional" and "edge" are OK for now at least. Both 3GPP and ETSI have published white papers using the "edge cloud" term. Maybe "regional" could be "central" or "core" if there is only one such cluster but I think "regional" is reasonable.

On "site-type", maybe that is a little restrictive in its scope. Would "cluster-type" or "workload-cluster-type" be better?

Reply:
If everyone is OK with it, let's leave it at "site-type"